student-cvs/alifm-2843/index.html

```diff
@@ -0,0 +1,311 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="/main.scss" />
+    <script src="https://cdn.jsdelivr.net/npm/dayjs@1/dayjs.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/dayjs@1/plugin/utc.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/dayjs@1/dayjs.min.js"></script>
+    <script
+    src="https://cdnjs.cloudflare.com/ajax/libs/dayjs/1.9.4/dayjs.min.js"
+    integrity="sha512-XZSHSEFj4QeE0G4pwy4tToyAhF2VXoEcF9CP0t1PSZMP2XHhEEB9PjM9knsdzcEKbi6GRMazdt8tJadz0JTKIQ=="
+    crossorigin="anonymous"
+    ></script>
+    <title>Alianna's web page</title>
+</head>
+
+<body>
+    <header>
+        <h1>AF</h1>
+
+        <nav class="topnav" id="myTopnav">
+            <ul>
+                <li>
+                    <a href="#">
+                        <i class="fa fa-fw fa-home"></i>
+                        Home
+                    </a>
+                </li>
+                <li>
+                    <a href="#about">
+                        <i class="fa fa-fw fa-user"></i>
+                        About me
+                    </a>
+                </li>
+                <li>
+                    <a href="#contact">
+                        <i class="fa fa-fw fa-envelope"></i>
+                        Contact
+                    </a>
+                </li>
+                <li>
+                    <a href="#acknowledgements">
+                        <i class="fa fa-fw fa-trophy"></i>
+                        Acknowledgements
+                    </a>
+                </li>
+             
+                <li>
+                    <a href="javascript:void(0);" class="icon">
```

Comment:
```suggestion
                    <a href="#" class="icon">
```

can be replaced with a simple "#"